docs/src/main/sphinx/connector/hive.rst

```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or
+administrative tasks, such as flushing the metadata caches::
+
+    CALL system.flush_metadata_cache()
```

Comment:
I don't think this procedure should be spotlighted as the first example.
I'd rather have users not need to use it.

Reply:
You could use a fake name here. `system.example_procedure()`

Reply:
Updated to a more generic example.